src/Auth/CookieTokenAuthenticate.php

```diff
@@ -257,7 +257,7 @@ public function afterIdentify(Event $event, array $user, BaseAuthenticate $auth)
             return;
         }
 
-        $cookieTokenComponent = $getCookieTokenComponent();
+        $cookieTokenComponent = $cookieTokenComponent = $this->getCookieTokenComponent();
```

Comment:
Well-spotted! Though this change accidentally assigns `$cookieTokenComponent` twice. It should be:

```php
$cookieTokenComponent = $this->getCookieTokenComponent();
```

Could you make a new commit to fix this?

Reply:
Fixed.